lib/system.mli

```diff
@@ -58,6 +58,11 @@ val is_in_system_path : string -> bool
 val where_in_path :
   ?warn:bool -> CUnix.load_path -> string -> CUnix.physical_path * string
 
+(** [get_output_path fname] turns fname into an absolute name, if not
+    already the case, using the default output-directory or the
+    current directory, if the default output-directory is not set *)
```

Comment:
IIUC the current code if given a relative -output-directory will produce a relative path here. We should absolutify the argument or change this comment (probably absolutify, making it robust to `Cd`).

Reply:
Right. Maybe should we make the output directory absolute at the time of parsing the command line then. Otherwise, if we do `Cd`, it will dynamically modify the output directory, which is probably not be what we want.

Unless you have a better idea, I'll do that.

Reply:
Actually, the same question arises for options `-native-output-dir` and `-nI` (native_include_dirs).

Reply:
"Otherwise, if we do Cd"... (sorry, I was repeating what you had said).